doc/py_tutorials/py_setup/py_setup_in_ubuntu/py_setup_in_ubuntu.markdown

```diff
@@ -86,12 +106,11 @@ But it may be a little old.
 If you want to get latest libraries, you can install development files for system libraries of these formats.
 
 ```
-sudo apt-get install libpng-devel
-sudo apt-get install libjpeg-turbo-devel
-sudo apt-get install jasper-devel
-sudo apt-get install openexr-devel
-sudo apt-get install libtiff-devel
-sudo apt-get install libwebp-devel
+sudo apt-get install libpng-dev
+sudo apt-get install libjpeg-turbo8-dev
```

Comment:
`libjpeg-dev` is [enough](https://packages.ubuntu.com/bionic/libjpeg-dev) (aliased to underlying package)